hash/risc0/bench/src/benches/poseidon2_bn128.rs

```diff
@@ -0,0 +1,97 @@
+use methods::{
+    POSEIDON2_BN128_ELF, POSEIDON2_BN128_ID
+};
+use risc0_zkvm::{default_prover, ExecutorEnv};
+use zkhash::{fields::{bn256::FpBN256, utils::random_scalar}/* , poseidon2::poseidon2_instance_bn256::POSEIDON2_BN256_PARAMS*/};
+// use zkhash::poseidon2::poseidon2::Poseidon2;
+// use std::convert::TryFrom;
+use std::time::Instant;
+// use zkhash::merkle_tree::merkle_tree_fp::MerkleTree;
+// use std::convert::TryInto;
+// use hex::encode_to_slice;
+use ark_serialize::{CanonicalSerialize, CanonicalDeserialize};
+
+
+pub fn poseidon2_bn128_bench(mt_depth: usize) {
+    
```

Comment:
this either runs very long time or forever on my machine. Even with tree depth=1, it didn't finish

Reply:
yes. Poseidon2 on bn128 takes around ~1400seconds for a tree of depth 4 and ~700 seconds for tree of depth 2.